acra-censor/handlers/blacklist_handler.go

```diff
@@ -45,21 +46,24 @@ func (handler *BlacklistHandler) CheckQuery(query string) (bool, error) {
 				case *sqlparser.AliasedTableExpr:
 					err = handler.handleAliasedTables(fromStatement.(*sqlparser.AliasedTableExpr))
 					if err != nil {
-						log.WithError(err).Debugln("error from BlacklistHandler.handleAliasedTables")
+						log.WithError(err).Debugln("acra-censor: error from BlacklistHandler.handleAliasedTables")
```

Comment:
instead using prefix like `acra-censor` better to create one logger with predefined field like `service_name` and use it across all code of module (like suggested @vixentael )